index.h

```diff
@@ -173,19 +183,29 @@ const char MAIN_page[] PROGMEM = R"=====(
                     <br /> <br />
                     <label for="status">Additional Comments</label><br />
                     <textarea class="textbox comments" name="message" id="commentsInput" cols="30" rows="2"></textarea><br />
-                    <input type="submit" class="sendReportBtn" value="SEND REPORT" />
+                    <div id="count">
+                    
+                    </div>
+                    <input type="submit"  class=" button-on" id="sendReportBtn" value="SEND REPORT" />
```

Comment:
May or may not be an issue, but you have an extra space inside the quotes here: `class=" button-on"`

Reply:
will fix